testutil/rand.go

```diff
@@ -0,0 +1,17 @@
+package testutil
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/cryptorand"
+)
+
+// MustString returns a random string of length n.
+func MustString(t *testing.T, n int) string {
```

Comment:
Perhaps `MustRandString`? I feel that in this case, neither package nor func name help describe what it does. Without context, I would expect it to either verify or convert the input as a string 😄

Reply:
Done!